scala3doc/src/dotty/dokka/compat.scala

```diff
@@ -28,6 +28,12 @@ type JMap[K, V] = java.util.Map[K, V]
 type JHashMap[K, V] = java.util.HashMap[K, V]
 type JMapEntry[K, V] = java.util.Map.Entry[K, V]
 
+private val emptyListInst = JList()
```

Comment:
Can we preemptively use `Collections.emptyList` here? I'd prefer to avoid issues due to some code modifying our "empty" list.